### DISCUSSION TITLE
Unable to execute MVC sandbox project.

### DISCUSSION THREAD

**Discussion by topagarwal** [dotnet/aspnetcore#24353] [General]
I got latest AspNet core from github and installed all the dependencies as mentioned in the wiki,
I am able to build the MVC solution by running build.cmd from within the src\MVC project.
However, when I open .slnf file using startvs.cmd command and set MVCSandbox as the startup project (not using IIS), i get following error:
<pre>
C:\AspNetCoreSrcCode\src\Razor\Microsoft.NET.Sdk.Razor\src\build\netstandard2.0\Microsoft.NET.Sdk.Razor.ScopedCss.targets(71,3): error MSB4062: The "Microsoft.AspNetCore.Razor.Tasks.DiscoverDefaultScopedCssItems" task could not be loaded from the assembly C:\AspNetCoreSrcCode\artifacts\bin\Microsoft.NET.Sdk.Razor\Debug\sdk-output\tasks\net46\Microsoft.NET.Sdk.Razor.Tasks.dll.  
Confirm that the  declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.
</pre>

Using windows 10 and installed all extensions and sdk as per installVS script in eng/scripts.
It will be great if someone can help me understand what I might be missing here.

Thanks.

**Comment by dougbu** [comment]
@mkArtakMSFT as @topagarwal asked in https://github.com/dotnet/aspnetcore/issues/24270#issuecomment-664722878, please let them know if this should be filed under issues.

**Reply by topagarwal** [reply]
I got sandbox project working.
Did git clean  - src\Mvc> git clean -dfx -e .dotnet/ -e .tools/ -e .vs/ -e .vscode/
and then executed build.cmd from commandling in powershell.
Openend Visual studio startvs.cmd MVC.slnf
Set the Sandbox project as startup project and
Run it.

Earlier, after building from command line, I was again rebuilding the project in VS and that caused issue.
I am unblocked for now but unsure how things will work when I make changes to sandbox project to test few things.
